aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -912,14 +998,22 @@ mod tests {
 
         let mut base_update = BTreeMap::new();
         let mut additional_update = BTreeMap::new();
-        base_update.insert(key.clone(), GroupWrite {
-            metadata_op: write_op_with_metadata(0, 100), // create
-            inner_ops: BTreeMap::new(),
-        });
-        additional_update.insert(key.clone(), GroupWrite {
-            metadata_op: write_op_with_metadata(2, 200), // delete
-            inner_ops: BTreeMap::new(),
-        });
+        base_update.insert(
+            key.clone(),
+            GroupWrite::new(
+                write_op_with_metadata(CREATION, 100), // create
+                vec![],
+                0,
+            ),
+        );
+        additional_update.insert(
```

Comment:
It this the new linter? I've seen a lot of divergence from nightly and the one it uses in my editor, so just making sure it's not the same for you @vusirikala

Reply:
Yes. It's the linter.

Reply:
I now do cargo +nightly fmt before I push, but it's annoying :) didn't realize other folks have the same issue as me not being able to use nightly in the editor. anyway, checks should not allow landing it, so you can fix whenever it's convenient for you.